CMakeLists.txt

```diff
@@ -2,7 +2,7 @@
 #   intro: https://codingnest.com/basic-cmake/
 #   best practices (3.0+): https://gist.github.com/mbinna/c61dbb39bca0e4fb7d1f73b0d66a4fd1
 
-cmake_minimum_required(VERSION 2.8.12)
+cmake_minimum_required(VERSION 3.9)
```

Comment:
Is there a way to use 2.8.12? We need to be able to build on as many systems as possible.